source/sass/blocks/main-index/download-app.scss

```
@@ -0,0 +1,80 @@
+.download-app {
+  padding-top: 34px;
+
+}
+
+.download-app__slogan {
+  margin: 0 0 34px 0;
+
+  font-weight: 300;
+  font-size: 24px;
+  line-height: 29px;
+  color: $font-color-secondary;
+  text-align: center;
+}
+
+.download-app__wrapper {
+  padding-top: 28px;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-bottom: 38px;
+
+  background-color: $font-color-primary;
+
+}
+
+.download-app__description{
+  margin: 0;
+  padding: 0;
+
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 24px;
+  color: $font-color-secondary;
+
+}
+
+.download-app__list {
+  display: flex;
+  align-items: center;
+  margin-top: 32px;
+  margin-bottom: 28px;
+  padding: 0;
+  list-style-type: none;
+
+}
+
+.download-app__item {
+  margin-right: 35px;
+}
+
+.download-app__item:last-child {
+  margin-right: 0px;
```

Нули у нас по правилам оформления пишутся без единиц измерения:
```suggestion
  margin-right: 0;
```